index.html

```diff
@@ -0,0 +1,54 @@
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet" />
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
+    </head>
+    <body>
+       
+             <section class="header">
+                <div class="place">
+                    <a href="#"><h3 class="name">John Doe</h3></a>
+
+                 </div>
+               <div class="menu">
+                <a href="#"><svg xmlns="http://www.w3.org/2000/svg" class="bar" width="1.5em" height="2em" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24"><path fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16"/></svg></a>
+               </div>
+    
+            </section>
+             <section class="headlines">
+                    <p class="primaryText">Hey There.I’m Munana</p>
+                   <p class="supportText">I’am a Software Developer</p>
+                    <p class="description" >I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.</p>
+
+                 
+                 <div class="socialmedia">
+                 
+                   <a href="https://github.com/clin2on3mun" target="https://github.com"><img src="images/C.png" height="17" alt="github"/></a>
+                   <a href="https://www.linkedin.com/in/munana-clinton-b6122720b/"><svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24"><g   stroke-linecap="round" stroke-linejoin="round" stroke-width="1" stroke-height="4"><path d="M16 8a6 6 0 0 1 6 6v7h-4v-7a2 2 0 0 0-2-2a2 2 0 0 0-2 2v7h-4v-7a6 6 0 0 1 6-6zM2 9h4v12H2z"/><circle cx="4" cy="4" r="2"/></g></svg></a>
+                   <a href=""><img src="images/hi.png" height="17" alt="peace" ></a>
+                   <a href="https://twitter.com/ClintonMunana"><svg xmlns="http://www.w3.org/2000/svg" class="tweet"width="1em" height="1em" preserveAspectRatio="xMidYMid meet" viewBox="0 0 15 15"><path fill="currentColor" d="M14.977 1.467a.5.5 0 0 0-.87-.301a2.559 2.559 0 0 1-1.226.763A3.441 3.441 0 0 0 10.526 1a3.539 3.539 0 0 0-3.537 3.541v.44C3.998 4.75 2.4 2.477 1.967 1.325a.5.5 0 0 0-.916-.048C.004 3.373-.157 5.407.604 7.139C1.27 8.656 2.61 9.864 4.51 10.665C3.647 11.276 2.194 12 .5 12a.5.5 0 0 0-.278.916C1.847 14 3.55 14 5.132 14h.048c4.861 0 8.8-3.946 8.8-8.812v-.479c.363-.37.646-.747.82-1.236c.193-.546.232-1.178.177-2.006Z"/></svg></a> 
+                   <a href="#"><img src="images/M.png" height="15" alt="M"></a>
+                </div>
```

Comment:
- Kindly, when you list social media use `ul` to the group The list of icons and `li` per icon; each icon has a link (`a` tag) because `a` tag will let you connect your social networks with the icons, using href.👍👌
- Please be sure that all social icon images are displayed correctly.👍👌